forge/test/mlir/test_ops_tf.py

```diff
@@ -149,12 +153,18 @@ def test_maxpool2d(
     class MaxPool(tf.keras.Model):
         def __init__(self):
             super().__init__() 
+            if pre_conv:
```

Comment:
Looks good! Let's add cases with pre/post eltwise ops as well, just to include some ops that don't have special input handling, and that are not bound to common use cases like conv + pool is.

Reply:
Actually, that isn't working so smoothly. Issue here: https://github.com/tenstorrent/tt-mlir/issues/738